deltachat-ios/Controller/QrCodeReaderController.swift

```diff
@@ -95,32 +95,37 @@ class QrCodeReaderController: UIViewController {
         guard let connection = videoPreviewLayer.connection else {
             return
         }
-
         guard connection.isVideoOrientationSupported else {
             return
         }
-
         let statusBarOrientation = UIApplication.shared.statusBarOrientation
         let videoOrientation: AVCaptureVideoOrientation =  statusBarOrientation.videoOrientation ?? .portrait
 
         if connection.videoOrientation == videoOrientation {
             print("no change to videoOrientation")
             return
         }
-
         videoPreviewLayer.frame = view.bounds
         connection.videoOrientation = videoOrientation
         videoPreviewLayer.removeAllAnimations()
     }
 
     // MARK: - actions
     func startSession() {
+        #if targetEnvironment(simulator)
+        // ignore if run from simulator
+        #else
         captureSession.startRunning()
-    }
+        #endif
+        }
 
     func stopSession() {
         captureSession.stopRunning()
     }
+
+    @objc private func cameraAccessButtonPressed(_ sender: UIButton) {
+        print("Camera button pressed")
+    }
```

Comment:
why is that overided now? how is that related to simulator or not?

Reply:
forgot to delete.